source/auth/auth.rst

```diff
@@ -420,9 +540,9 @@ Auth Related Options
 --------------------
 
 authMechanism
-	MONGODB-CR, MONGODB-X509, GSSAPI, PLAIN, SCRAM-SHA-1
+	MONGODB-CR, MONGODB-X509, GSSAPI, PLAIN, SCRAM-SHA-1, SCRAM-SHA-256
 
-	Sets the Mechanism property on the MongoCredential. The default is MONGODB-CR if <= 2.6, otherwise SCRAM-SHA-1.
+	Sets the Mechanism property on the MongoCredential. The default is determined based on server version and user credentials.
```

Comment:
"server version, user credentials, and the saslSupportedMechs field of the server's IsMaster reply"